apps/android/app/src/main/res/values-ru/strings.xml

```diff
@@ -0,0 +1,218 @@
+    <string name="profile_will_be_sent_to_contact_sending_link">Ваш профиль будет отправлен контакту, от которого Вы получили эту ссылку.</string>
+    <string name="connect_via_link_verb">Соединиться</string>
+
+    <!-- Server info - ChatModel.kt -->
+    <string name="server_connected">Соединен с сервером</string>
+    <string name="server_connecting">Соединяется с сервером…</string>
+    <string name="connected_to_server_to_receive_messages_from_contact">Установлено соединение с сервером, через который Вы получается сообщения от этого контакта.</string>
+    <string name="trying_to_connect_to_server_to_receive_messages_with_error">Устанавливается соединение с сервером, через который Вы получаете сообщения от этого контакта (ошибка: <xliff:g id="errorMsg">%1$s</xliff:g>).</string>
+    <string name="trying_to_connect_to_server_to_receive_messages">Устанавливается соединение с сервером, через который Вы получаете сообщения от этого контакта.</string>
+
+    <!-- Item Content - ChatModel.kt -->
+    <string name="deleted_description">удалено</string>
+    <string name="sending_files_not_yet_supported">отправка файлов не поддерживается</string>
+    <string name="receiving_files_not_yet_supported">получение файлов не поддерживается</string>
+    <string name="sender_you_pronoun">Вы</string>
+    <string name="unknown_message_format">неизвестный формат сообщения</string>
+    <string name="invalid_message_format">ошибочный формат сообщения</string>
+
+    <!-- SMP Server Information - SimpleXAPI.kt -->
+    <string name="error_saving_smp_servers">Ошибка при сохранении SMP серверов</string>
+    <string name="ensure_smp_server_address_are_correct_format_and_unique">Пожалуйста, проверьте, что адреса SMP серверов имеют правильный формат, каждый адрес на отельной строке и не повторяется.</string>
+
+    <!-- API Error Responses - SimpleXAPI.kt -->
+    <string name="contact_already_exists">Существующий контакт</string>
+    <string name="you_are_already_connected_to_vName_via_this_link">Вы уже соединены с <xliff:g id="contactName" example="Alice">%1$s!</xliff:g> через эту ссылку.</string>
+    <string name="invalid_connection_link">Ошибка в ссылке контакта</string>
+    <string name="please_check_correct_link_and_maybe_ask_for_a_new_one">Пожалуйста, проверьте, что Вы использовали правильную ссылку, или попросите, чтобы Ваш контакт отправил Вам новую.</string>
+    <string name="cannot_delete_contact">Невозможно удалить контакт!</string>
+    <string name="contact_cannot_be_deleted_as_they_are_in_groups">Контакт <xliff:g id="contactName" example="Jane Doe">%1$s!</xliff:g> не может быть удален, это член групп(ы) <xliff:g id="groups" example="[team, chess club]">%2$s</xliff:g>.</string>
+    <string name="icon_descr_instant_notifications">Мгновенные уведомления</string>
+
+    <!-- background service notice - SimpleXAPI.kt -->
+    <string name="private_instant_notifications">Безопасные мгновенные уведомления!</string>
```

Comment:
Or "конфиденциальные"? Otherwise it would imply other types of notifications (i.e. if we add push notifications to android) are "non secure"
```suggestion
    <string name="private_instant_notifications">Приватные мгновенные уведомления!</string>
```